webapi/src/test/scala/org/knora/webapi/routing/admin/ProjectsRouteZSpec.scala

```diff
@@ -192,4 +193,22 @@ object ProjectsRouteZSpec extends ZIOSpecDefault {
         } yield assertTrue(body.contains(shortname))
       }
     )
+
+  val deleteProjectSpec =
```

Comment:
I guess you will have to add this test to a `suite` otherwise it will be ignored.

Reply:
thanks, done ;)